tests/testSaturationCorrection.py

```diff
@@ -21,24 +21,22 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-import os
 import unittest
 
-import lsst.utils.tests as tests
+import lsst.utils.tests
 import lsst.afw.image as afwImage
 import lsst.afw.geom as afwGeom
 import lsst.ip.isr as ipIsr
 
 
-class IsrTestCases(unittest.TestCase):
+class IsrTestCases(lsst.utils.tests.TestCase):
 
     def testSaturation(self):
         saturation = 1000
 
         bbox = afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Point2I(19, 19))
         maskedImage = afwImage.MaskedImageF(bbox)
         maskedImage.set(100, 0x0, 1)
-        exposure = afwImage.ExposureF(maskedImage, None)
```

Comment:
This line (and the other removed one) was flagged as unused. @timj found the related commit.